wikiHandler.js

```diff
@@ -48,11 +53,14 @@ async function fetchAndFormatAuctionData(auction_user, auction_contents, server)
         }
         fields.push(field);
     })
+
     return new Discord.MessageEmbed()
         .setColor(SERVER_COLOR[server])
-        .setTitle(`${auction_user} (${auction_mode})`)
+        .setTitle(`**${auction_mode}**`)
         .setDescription(`\`\`\`${auction_contents}\`\`\``)
-        .addFields(fields);
+        .addFields(fields)
+        .setFooter(`${auction_user} on Project 1999 ${utils.formatCapitalCase(server)}`)
+        .setTimestamp()
```

Comment:
Interesting that Timestamp still only shows the date on Desktop, but shows the date+time on mobile! Cool feature, should handle the complaints about mobile formatting not showing time easily I guess. :P

Reply:
The most annoying part of this is that on mobile it uses a bullet before the time and on desktop it uses a | .  But yeah, I think it'll be good.  It will also reliably format to user's local timezone which I'm not sure we would be able to do within our own code.